yaksh/models.py

```diff
@@ -1110,7 +1118,7 @@ def __str__(self):
 
 class StdioBasedTestCase(TestCase):
     expected_input = models.TextField(blank=True)
-    expected_output = models.TextField()
+    expected_output = models.TextField(blank=True)
 
     def get_field_value(self):
         return {"expected_output": self.expected_output,
```

Comment:
Just FYI, Can you please make sure that the expected input and expected output are forced to either a black string '' or a None when being assembled here or when being assembled into a JSON dict

Please also make sure that the evaluator is able to handle situations where expected input/output is black string
since None != '""